### DISCUSSION TITLE
Display and in-line math font sizes in Jupyterbook

### DISCUSSION THREAD

**Discussion by chipbrock** [executablebooks/meta#1191] [Q&A]
I cannot figure out how to separately control the size of mathematics in Jupyterbook, presumably through Mathjax. My presumption is that I can control this through mystyle.css, but that's not working. mystyle.css wouldn't attach, so I've pasted its contents below (trying to insert as code swallowed the CSS comment characters, and they matter here, so I've left it as plain text.). There are many different choices commented out, none of which had any effect as if mystyle.css is not in control of this aspect of HTML rendering. It is in control of regular text etc.

The consequence of what happens in the current and all previous attempts is that in-line sizes are fine, but display math is tiny. I've also attached a screen shot of what output looks like.

Assistance with this would be appreciated. 

![Untitled 6](https://github.com/executablebooks/meta/assets/4515277/78183767-aeb8-46ed-a3e6-0d290940faa8)

html {
font-family: "Palatino Linotype", "Book Antiqua", Palatino, serif;
/* 	font-family: sans-serif; */
/* 	font-family: sans-serif; */
  font-size: 14px;
    line-height: 1.5;
/* 	like newsprint */
    font-family: "PT Serif", 'Times New Roman', Times, serif;
	color: #1f0909;
	line-height: 1.5em;
}

body {
	background-color: #fcfcfb; !important;
}

<style>
strong { 
  font-weight: bold;
}
</style>

p.caption {
  font-size: 1.0rem;
  color: #777;
  margin-top: 10px;
}
p code {
  white-space: inherit;
}
pre {
  word-break: normal;
  word-wrap: normal;
}
pre code {
  white-space: inherit;
}

/*.MathJax { font-size: 1.1em !important; }*/

         
 	p {  font-family: "Palatino Linotype", "Book Antiqua", Palatino, serif;
    color: #1f0909;
    font-weight: 500;
    font-size: 1.4rem;
    line-height: 2.0rem;
    margin-top: 1.4rem;
    margin-bottom: 1.4rem;
    padding-right: 0;
    vertical-align: baseline;
}

/*.MathJax {
    font-size: 100% !important;
}*/

/*.math.inline .MathJax  {
  font-size: 1.5rem !important;
}

.math.display .MathJax  {
  font-size: 1.5rem !important;
  color: #b22222;
}*/

.MathJax_Display {
  margin: 5em 0em !important;
    color: #b22222;
}

.admonition.danger {
    font-size: 1.1rem;
     font-family: "Lucida Sans Unicode", "Lucida Grande", sans-serif;
/*	font-family: "Palatino Linotype", "Book Antiqua", Palatino, serif;*/
/*   color: #b94a48;*/
/*   background-color: #e0e8f3;*/
/*   border-color: #eed3d7;*/
}

div.admonition, div.topic, blockquote {
    clear: left;
         font-family: "Lucida Sans Unicode", "Lucida Grande", sans-serif;
}

/* -- topics ---------------------------------------------------------------- */

div.topic {
    border: 1px solid #ccc;
    padding: 7px;
    margin: 10px 0 10px 0;
         font-family: "Lucida Sans Unicode", "Lucida Grande", sans-serif;
}

p.topic-title {
    font-size: 1.1em;
    font-weight: bold;
    margin-top: 10px;
         font-family: "Lucida Sans Unicode", "Lucida Grande", sans-serif;
}

/* -- admonitions ----------------------------------------------------------- */

div.admonition {
    margin-top: 10px;
    margin-bottom: 10px;
    padding: 7px;
         font-family: "Lucida Sans Unicode", "Lucida Grande", sans-serif;
}

div.admonition dt {
    font-weight: bold;
         font-family: "Lucida Sans Unicode", "Lucida Grande", sans-serif;
}

p.admonition-title {
    margin: 0px 10px 5px 0px;
    font-weight: bold;
        font-size: 1.1rem;
         font-family: "Lucida Sans Unicode", "Lucida Grande", sans-serif;
}

/*div.body centered {*/
div.body p.centered {
    text-align: center;
    margin-top: 25px;
         font-family: "Lucida Sans Unicode", "Lucida Grande", sans-serif;
}

div.admonition > p {
    font-family: "Lucida Sans Unicode", "Lucida Grande", sans-serif;
     font-size: 1.1em;
     color: #686868;
      font-weight: 100;
     
}

blockquote.pull-quote,blockquote.epigraph,blockquote.highlights{
	font-size:1.25em;
	border-left:none;
	font-family: "Lucida Sans Unicode", "Lucida Grande", sans-serif;
     font-size: 1.0em;
	}
	
	blockquote p{
/*background-color:rgba(149,168,99,0.1);*/
/*border-left: 3px solid rgba(149,168,99,1.0);*/
/*     border-left: 5px solid rgba(182,83,84,1.0); */
/*    margin: 10px 0px 20px 0px;
    padding: 10px 0px 0px 40px;
      margin-left: 0px;
  padding-left: 60px; */
        font-size: 1.1rem;
        line-height: 1.2rem;  
        color: gray;
      font-family: Helvetica, sans-serif; 
/*	font-style: italic; */
  	}
  	
  	blockquote > p {
/*    padding: 15px;*/
    margin: 40px 0px;
      margin-left: 50px;
  margin-left: auto;
       padding-left: 15px;
                     padding-top: 25px;
              padding-bottom: 25px;
        background-color: #ffffff;
/*         border-left: 3px solid #ccc; */
/*    margin-left: 20px;*/
/*    padding-left: 20px;*/
font-size: 1.2rem;
line-height: 1.6rem;  
color: #212121;
    font-family: Helvetica, sans-serif; 
}
